### ISSUE TITLE
Issue with: (Advanced Settings, Output: X) when upgrading from old version 0.6.2 to 0.7.6

### ISSUE THREAD

**Issue by San3-Cod3** [Tenpi/Crunchyroll-Downloader#89]
Issue with: (Advanced Settings, Output: X) when upgrading from old version 0.6.2 to 0.7.6.

My settings were retained when I upgraded which is what I wanted, but now the 'Output:' setting line in Advanced Settings cannot be modified; can't interact with that setting line.

I want the 'Output:' setting line how it currently is, with the exception of wanting to remove the leading zero in front of {episodeNumber} which I added in the past - image below.

http://puu.sh/K0TXZ/adc3b3165d

Now, in addition to that, it would be nice if "{episodeNumber}" could be user modified to be universally uniform and consistent to their own preference, I.E having episode numbers use a preferred naming convention like 001, 01, 1, 0001.

Being able to choose the amount of leading zeroes to be consistent with user preference would be handy.

With my messy method, things like 012, 031 022 for example would happen, instead of 12, 31, 22.

Basically having control of: 
1 2 3 4 5 6 7 8 9
01 02 03 04 04 06 07 08 09
001 002 003 004 005 006 007 008 009
0001 0002 0003 0004 0005 0006 0007 0008 0009

Would be pretty nice - but I know the headache that can happen when it comes to leading zeroes in code, so it's up to you if that's feasible to do.

In-short: single, double, triple and quadruple digit {episodeNumber} user-specific naming conventions.  This would be helpful.
Examples:
One Piece: Eps. 0001-1093 | creating digit length uniformity
Hunter X Hunter Eps. 001-148 | creating digit length uniformity
Erased: Eps. 01-12 | creating digit length uniformity
Naruto Special: 1, 2, 3 | creating digit length uniformity

**Comment by Tenpi** [comment]
Decrease the size of the window the whole logo contains an invisible element where you can drag the window around but it basically overrides all other input

**Comment by San3-Cod3** [comment]
> Decrease the size of the window the whole logo contains an invisible element where you can drag the window around but it basically overrides all other input

I usually have the window reduced to the default smaller size, I just maximised it for screenshot visibility purposes - but no matter the window size, I cannot interact with the output setting line, and clicking and holding the line does allow the window to be dragged around, so there must be an element prevent me from interacting with it - it could be the length or size of my old settings that were saved and maybe there's a compatibility issue.

I do not know where the application saves my output settings to though, as I could back it up, and then delete it and see what happens.

**Comment by Tenpi** [comment]
it's in `C:\Users\name\AppData` look for crunchyroll-downloader in Local or Roaming and there should be a config.json file there. 

I can move it down where it's easier to click, does it still not work on 0.7.7 (latest version)?

**Comment by San3-Cod3** [comment]
Still the same issue on latest: 0.7.7.

I have not tried the manual settings edit yet in %appdata%, will try this next and get back.

**Comment by San3-Cod3** [comment]
Okay, I manually updated the config.json in %appdata%, and this did show the modified line in the app as I made it shorter for testing, but I still cannot interact with the output setting line.

**Comment by San3-Cod3** [comment]
After some additional fiddling around in regular window size and maximised, there are definitely some element blocking issues - in regular window size, the "Output:" setting line can't be interacted with - in maximised window size, more setting lines can't be interacted with, namely: "Output:", "Concurrent Downloads", "Video Quality:" and "Video Codec:"

**Comment by San3-Cod3** [comment]
Oh, and other than the above issue, downloading as a free user, and not logged in, fetching shows that aren't locked to Premium seem to be working fine.  Shows like Demon Slayer and Vinland Saga Season 2 as examples, are working nicely for downloading and playback for current testing purposes.  Very much appreciate you having updated this! The GUI is splendid :)

![ss (2024-02-14 at 09 53 54)](https://github.com/Tenpi/Crunchyroll-Downloader/assets/95466780/9656552f-b694-47dd-b9a0-c055ce0d62e5)

**Comment by Tenpi** [comment]
This was fixed in 0.7.8 also I added new templates to do what you asked

`{episodeNumber0}` - format like 02
`{episodeNumber00}` - format like 002
`{episodeNumber000}` - format like 0002
